### PR TITLE
Create while eating

### PR DESCRIPTION
**Pull request by SHEIKDAOUDABDULRAUF**
created comment before eating

### CODE REVIEW

File: while eating [hunk 1]
```diff
@@ -0,0 +1 @@
+itadakimasu!!!
```
Comment (Sheikdaoud) [inline while eating:1]: what is after eating?

Comment (Sheikdaoud) [reply while eating:1]: add that also

Comment (SHEIKDAOUDABDULRAUF) [reply while eating:1]: ok

Comment (SHEIKDAOUDABDULRAUF) [inline while eating:1]: arigato?

**Pull request merged**